docs/macos-11-onboarding.md

```diff
@@ -0,0 +1,26 @@
+# macOS 11 (Big Sur) onboarding guide
+
+macOS 11 pools were opened to existing customers only, and we are working on making them ready for production use, rebalancing capacity and analyzing possible load.
+
+Meanwhile, we'd like to start including customers to private preview on demand. 
+
+> Big Sur will automatically became available for current Enterprise customers. New enterprises and customers on other plans, please go through the sign up process described below.
```

Comment:
`current Enterprise customers` doesn't sound clear. May be say `for all Enterprise accounts created before N May`

Reply:
we don't know the exact date, so I just changed `current` to `existing` to match note in the main README